src/test/scala/io/github/pauljamescleary/petstore/infrastructure/endpoint/AuthTest.scala

```diff
@@ -0,0 +1,32 @@
+package io.github.pauljamescleary.petstore.infrastructure.endpoint
```

Comment:
break package statement apart to make imports below cleaner.

Reply:
Done